<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true" extracomment="Application Title">Watermark</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="Editor" name="editor">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>455</width>
         <height>297</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveFinish"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="actionReportIssue"/>
   </widget>
   <widget class="QMenu" name="menuPresets">
    <property name="title">
     <string>&amp;Presets</string>
    </property>
    <widget class="QMenu" name="menuSavedPreset">
     <property name="title">
      <string>&amp;Presets</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources/icons.qrc">
       <normaloff>:/icons/48px/presets</normaloff>:/icons/48px/presets</iconset>
     </property>
     <addaction name="action_4"/>
    </widget>
    <addaction name="actionSavePreset"/>
    <addaction name="separator"/>
    <addaction name="menuSavedPreset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPresets"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetWatermark">
   <property name="windowIcon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/watermark</normaloff>:/icons/48px/watermark</iconset>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>&amp;Watermark</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="WatermarkForm" name="watermarkForm"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetPreview">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowIcon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/preview</normaloff>:/icons/48px/preview</iconset>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Pre&amp;view</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="Preview" name="preview">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetCrop">
   <property name="windowIcon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/crop</normaloff>:/icons/48px/crop</iconset>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>C&amp;rop Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="CropForm" name="cropForm"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/exit</normaloff>:/icons/48px/exit</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/save</normaloff>:/icons/48px/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/save-as</normaloff>:/icons/48px/save-as</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
   <property name="toolTip">
    <string>Save the image as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/settings</normaloff>:/icons/48px/settings</iconset>
   </property>
   <property name="text">
    <string>Sett&amp;ings</string>
   </property>
   <property name="toolTip">
    <string>Open settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/help</normaloff>:/icons/48px/help</iconset>
   </property>
   <property name="text">
    <string>&amp;Documentation</string>
   </property>
   <property name="toolTip">
    <string>Show help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/about</normaloff>:/icons/48px/about</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About WaterMark</string>
   </property>
  </action>
  <action name="actionReportIssue">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/bug</normaloff>:/icons/48px/bug</iconset>
   </property>
   <property name="text">
    <string>Report &amp;issue</string>
   </property>
   <property name="toolTip">
    <string>Report an issue</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/open</normaloff>:/icons/48px/open</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open an image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionStartWatcher">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="actionSaveFinish">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/save</normaloff>:/icons/48px/save</iconset>
   </property>
   <property name="text">
    <string>Save &amp;finish</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/license</normaloff>:/icons/48px/license</iconset>
   </property>
   <property name="text">
    <string>&amp;License</string>
   </property>
  </action>
  <action name="actionSavePreset">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/48px/save-preset</normaloff>:/icons/48px/save-preset</iconset>
   </property>
   <property name="text">
    <string>&amp;Save preset</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string notr="true">...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WatermarkForm</class>
   <extends>QWidget</extends>
   <header>watermarkform.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CropForm</class>
   <extends>QWidget</extends>
   <header>cropform.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Editor</class>
   <extends>QWidget</extends>
   <header>editor.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Preview</class>
   <extends>QWidget</extends>
   <header>preview.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
